resources/authgear/templates/en/web/__toc_pp_footer.html

```diff
@@ -1,7 +1,26 @@
 {{ define "__toc_pp_footer.html" -}}
-{{ if ($.Translations.HasKey "toc-pp-footer") }}
+{{ $showFooter := $.Translations.HasKey "toc-pp-footer" }}
+{{ $termsOnly := $.Translations.HasKey "terms-of-service-link" }}
+{{ $privacyOnly := $.Translations.HasKey "privacy-policy-link" }}
+
+{{ $variant := "" }}
+{{ if and ($termsOnly) (not $privacyOnly) }}
+  {{ $variant = "termsOnly" }}
+{{ end }}
+{{ if and ($privacyOnly) (not $termsOnly) }}
+  {{ $variant = "privacyOnly" }}
+{{ end }}
+{{ if and ($termsOnly) ($privacyOnly) }}
+  {{ $variant = "both" }}
+{{ end }}
+{{ if and (not $termsOnly) (not $privacyOnly) }}
+  {{ $showFooter = false }}
+{{ end }}
+
 <p class="text-sm primary-txt m-0">
-{{ template "toc-pp-footer" (dict "termsOfService" (.Translations.RenderText "terms-of-service-link" nil) "privacyPolicy" (.Translations.RenderText "privacy-policy-link" nil)) }}
+  {{ template "toc-pp-footer" (dict "showFooter" $showFooter
```

Comment:
showFooter should be used in a if statement. In this current usage, we will render an empty p.

Reply:
@louischan-oursky updated :adore2x: